tests/utils/log/test_es_task_handler.py

```diff
@@ -262,21 +277,55 @@ def test_set_context_w_json_format_and_write_stdout(self):
         self.es_task_handler.json_format = True
         self.es_task_handler.set_context(self.ti)
 
-    def test_close(self):
-        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
-        self.es_task_handler.formatter = formatter
+    def test_close_with_log_id(self):
+        es_task_handler = ElasticsearchTaskHandler(
+            self.local_log_location,
+            self.filename_template,
+            self.log_id_template,
+            self.end_of_log_mark,
+            self.write_stdout,
+            True,  # json_format
```

Comment:
can you also have another test with non json format? and set the formatter as `logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')`

Reply:
when you work with elastic, json formatter is the way to go. We do not have grok filter in the unit tests to parse and index this non-json format message and then index them and look up as I do in the test case here. Realistically we should probably remove the non-json format option, and it is not practical to deploy it correctly as far as I can tell.